<launch>
  <!-- Giao tiếp với traffic -->
  <!-- <node name="stiClient" pkg="agvtraffic_simulation" type="stiClient.py"/> -->

  <!-- Nhận lộ trình và lỗi từ stiControl_fake -->
  <!-- <node name="stiClient1_fake" pkg="agvtraffic_simulation" type="stiClient.py">
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot1.yaml" />
  </node>   -->

  <node name="stiDevice1_fake" pkg="agvtraffic_simulation" type="stiDevice_fake.py" >
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot1.yaml" />
  </node>

  <!-- Điều phối chung -->
  <node name="stiControl1_fake" pkg="agvtraffic_simulation" type="stiControl_fake.py" output="screen">
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot1.yaml" />
  </node>

  <!-- Điều hướng AGV  -->
  <!-- <node name="goal1_control" pkg="agvtraffic_simulation" type="goal_control.py">
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot1.yaml" />
  </node> -->
  
  <node name="goal1_control_fake" pkg="agvtraffic_simulation" type="goal_control_fake.py">
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot1.yaml" />
  </node> 

  <!-- Vị trí của AGV trên Rviz -->
  <node name="agv1_sim" pkg="agvtraffic_simulation" type="agv_sim.py" respawn="false" >
    <rosparam command="load" file="$(find agvtraffic_simulation)/config/param_robot1.yaml" />
  </node>

</launch>